app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<p style="color: green"><%= notice %></p>
+<div class="card-header container text-center">
+  Recipes
+  <%= link_to "Add recipe", new_recipe_path, class: 'btn btn-dark btn-sm' %>
+</div>
+<% @recipes.each do |recipe| %>
+  <br>
+  <div class="container card mb-3" style="max-width: 540px;">
+  <div class="row g-0">
+    <div class="col-md-4 p-1">
+      <%= link_to recipe_path(recipe.id), class: 'btn btn-warning p-2 mt-1' do %>
+      <h1><%= recipe.name %></h1>
+      <% end %>
+      <%= button_to "REMOVE", recipe, method: :delete, class:"btn btn-sm" %>
```

Comment:
- [ ] Recipe is not being deleted and no error message is being displayed. Please make sure that only recipe owners can `delete`.

Reply:
Sorted in the side https://github.com/kwambiee/rails-recipe-app/pull/33